docs/src/index.md

```diff
@@ -60,6 +60,14 @@ Currently, for likelihoods `TuringGLM.jl` supports:
 
 Take a look at the tutorials for all supported likelihood and models:
 
+* [Linear Regression](tutorials/linear_regression)
+* [Logistic Regression](tutorials/logistic_regression)
+* [Poisson Regression](tutorials/poisson_regression)
+* [Negative Binomial Regression](tutorials/negativebinomial_regression)
+* [Robust Student-t Regression](tutorials/robust_regression)
+* [Hierarchical Models](tutorials/hierarchical_models)
+* [Custom Priors](tutorials/custom_prior)
+
```

Comment:
```suggestion
```

Reply:
There's a risk that the list becomes outdated at some point. In the browser on a screen with at least 700px or so, the tutorials are clearly visible at the left hand side.